README.md

```diff
@@ -35,14 +47,29 @@ router.get('/', function (req, res) {
 });
 ```
 
-`index.html`
-```html
-<html>
-	<head></head>
-	<body>
-		<div id="app"><%- content %></div>
-	</body>
-</html>
+### Option: Pass jsx
+
+`index.js`:
+
+```javascript
+var router = require('express')();
+var reactExpressMiddleware = require('react-express-middleware');
+var PageWrapper = require('./page-wrapper.jsx');
```

Comment:
For examples, I like to keep it simple, so maybe don't show any component's, just use divs.